model-integration/src/main/java/ai/vespa/llm/clients/LocalLLM.java

```diff
@@ -31,7 +34,7 @@
  *
  * @author lesters
  */
-public class LocalLLM extends AbstractComponent implements LanguageModel {
+public class LocalLLM extends AbstractComponent implements LanguageModel, Generator {
```

Comment:
The role of Generator is to apply a LanguageModel to a generation task specified by a context. This is a more general API so it's a bit messy for it to know about that context (and also to call itself through a utility defined elsewhere). I see the purpose though ... maybe there's a better solution, but I need to get some breakfast now.